canonical-web-and-design/vanilla-framework
# Fix button width in mobile nav

## Done

- Add styles to fix `p-navigation__link` when used on a button element

Fixes #4444 

## QA

- Open [demo](insert-demo-url)
- Go to the default Navigation example
- Inspect the list of links, and add another `p-navigation__link` on a button element, wrapped in a `<li class="p-navigation__item"></li>`
- Check the button is full width, compared to on live or [here](https://codepen.io/huwshimi/pen/qBxWmLj)


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
